Erwa on Cowboy-2.x version

ethrbh opened the pull request:
hello,

I made some changes in Erwa for able to work it on Cowboy-2.x version. Please have look at the changes, and apply it to the main repo if you do not see any problem.

thanks for your help
/Robi

ethrbh committed:
Update Lager dependency with the newest version

ethrbh committed:
Update Lager dependency with the newest version

ethrbh committed:
Eliminate compilation warnings, by comment the unused variables

ethrbh committed:
Eliminate compilation warnings, by comment the unused variables

ethrbh committed:
Erwa can works with Cowboy-2.x version

ethrbh committed:
Erwa can works with Cowboy-2.x version

ethrbh committed:
Use Cowboy master branch by default

ethrbh committed:
Use Cowboy master branch by default

ethrbh committed:
Update .app.src file to be sync with Makefile

- Cowboy and ranch projects are needed but these were set in Makefile
only, and .app.src was not updated. Because this, erlang.mk does not
packed these two app into the release package.

ethrbh committed:
Update .app.src file to be sync with Makefile

- Cowboy and ranch projects are needed but these were set in Makefile
only, and .app.src was not updated. Because this, erlang.mk does not
packed these two app into the release package.

ethrbh committed:
Update for able to work with Cowboy-2.x version

Also, elogger is added as dependency

ethrbh committed:
Update for able to work with Cowboy-2.x version

Also, elogger is added as dependency

ethrbh committed:
Remove debug printouts

ethrbh committed:
Remove debug printouts

ethrbh committed:
Update Cowboy dependency

- new path of the app: dep_cowboy_commit = master

ethrbh committed:
Update Cowboy dependency

- new path of the app: dep_cowboy_commit = master

ethrbh committed:
Update dependencies list of ERWA

-ranch Erlang app has been removed from the list of dependencies,
because this is already a dependencie of Cowboy app.

ethrbh committed:
Update dependencies list of ERWA

-ranch Erlang app has been removed from the list of dependencies,
because this is already a dependencie of Cowboy app.

ethrbh committed:
Bug fix the previously made typo fault

-the wamper application was named wrongly in the dependencie list

ethrbh committed:
Bug fix the previously made typo fault

-the wamper application was named wrongly in the dependencie list

ethrbh committed:
Update for Erlang/OTP-20.0

"crypto:rand_uniform/2 is deprecated and will be removed in a future
release; use rand:uniform/1"

ethrbh committed:
Update for Erlang/OTP-20.0

"crypto:rand_uniform/2 is deprecated and will be removed in a future
release; use rand:uniform/1"

ethrbh commented:
hello Bash,

Please have a look at the changes I did in Erwa. If these are fine for you please merge into the main repo.

thanks,
/Robi

ethrbh commented:
hello Bash,

Sorry for disturbing you again. I am interesting, are you still working on Erwa project? If so, could you please have a look the changes I made for newer Erlang/OTP version?

thanks a lot,
/Robi

bwegh commented:
hi Robert,
I moved to cargotube.

can you please give it a try.
the latest master should provide basic wamp features.

which features do you use /need?

thanks, Bas

ethrbh commented:
hello Bas,

Thanks for your answer. Regarding to ERWA, I am using few things only. One realm what I have and using erwa_in_tcp and erwa_in_ws for cowboy web server. What do you think, how hard move from erwa to cargotube? I checked the project, but I did not see such examples what I probably can use for getting some help. Do you have maybe what is not pushed to the Git repo?

thanks,
/Robi

ethrbh committed:
Update to use lager dep from its original repo

ethrbh committed:
Update to use lager dep from its original repo

ethrbh committed:
Merge pull request #1 from ethrbh/use_lager_from_erlang-lager

Update to use lager dep from its original repo

ethrbh committed:
Merge pull request #2 from ethrbh/sync_with_original_master_branch_from_bwegh

Sync with original master branch from bwegh

ethrbh committed:
Update to use wamper from ethrbh

ethrbh committed:
Merge pull request #3 from ethrbh/dev_branch

Update to use wamper from ethrbh

ethrbh committed:
Update to use erlang-lager/lager 3.9.2

This update is required if Erlang OTP-24.x
is used. Otherwise Erwa compilation error
occures.

ethrbh committed:
Merge pull request #4 from ethrbh/dev_branch

Update to use erlang-lager/lager 3.9.2
